roles/ceph-client/defaults/main.yml

```diff
@@ -1,18 +0,0 @@
----
```

Comment:
All these symlinks shouldn't be necessary. As long as ``ceph-defaults`` is included in the same play as another role then that role should inherit all of the config in ``ceph-defaults/defaults/main.yml``.

These could be done by making ``ceph-defaults`` a dependency for the other roles or just including it directly in a playbook like:

```
- hosts: all
  roles:
     - ceph-defaults
     - ceph-common
     - ceph-mon
```

In that example ``ceph-common`` and ``ceph-mon`` should have access to all configuration set in ``ceph-defaults``.